src/Microsoft.OpenApi.Readers/Document.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.OpenApi.Readers
+{
+    /// <summary>
+    /// Loads an OpenApiDocument instance through Load/LoadAsync/Parse pattern
+    /// </summary>
+    public static class Document
```

Comment:
I'm not sure why those methods aren't defined on OpenAPIDocument directly?

Reply:
With the current implementation, since the `OpenApiDocument` class resides in the `Microsoft.OpenApi` project, defining the methods within the class would need us to take a dependency on the `Microsoft.OpenApi.Readers` library(for access to the StreamReader, TextReader etc) which already depends on the `Microsoft.OpenApi` lib. This will end up causing a circular dependency between the two.

Reply:
right, thanks for clarifying.